action.yml

```diff
@@ -50,8 +50,8 @@ runs:
       id: action
       shell: powershell
       run: |
-        $secure_password = ConvertTo-SecureString -String ${{ inputs.service-account-password }} -AsPlainText -Force
-        $secure_cert_password = ConvertTo-SecureString -String ${{ inputs.website-cert-password }} -AsPlainText -Force
+        $secure_password = ConvertTo-SecureString -String '${{ inputs.service-account-password }}' -AsPlainText -Force
+        $secure_cert_password = ConvertTo-SecureString -String $'{{ inputs.website-cert-password }}' -AsPlainText -Force
```

Comment:
Single quotes included in the password will still break this.

Reply:
See https://github.com/im-open/register-windows-service/pull/5 for an alternative.

Reply:
The `$Env:<var>` still has the same escape/encoding problem.  I did find a way to encapsulate using `@'` that works well.